README.rst

```diff
@@ -69,24 +69,26 @@ Follow the setup instructions of `openwisp-controller
 
     INSTALLED_APPS = [
         # django apps
-        # openwisp2 admin theme (must be loaded here)
-        'openwisp_utils.admin_theme',
         # all-auth
         'django.contrib.sites',
         'allauth',
         'allauth.account',
         'allauth.socialaccount',
         'django_extensions',
+        'django_filters',
         # openwisp2 modules
         'openwisp_users',
         'openwisp_controller.pki',
         'openwisp_controller.config',
         'openwisp_controller.connection',
         # monitoring
-        'notifications',
         'openwisp_monitoring.monitoring',
         'openwisp_monitoring.device',
         'openwisp_monitoring.check',
+        # notifications
+        'openwisp_notifications',
+        # openwisp2 admin theme (must be loaded here)
+        'openwisp_utils.admin_theme',
```

Comment:
can you quickly explain why this change is needed? I guess is for the template but just double checking.

Reply:
Yes, it is for template. We are now using `base_site.html` instead of `base.html` in `openwisp-notifications`, hence reordering of installed apps was required. :smile:

Reply:
Ok, please remove `apptemplates.Loader` from `settings.py` (it should fail now that django-apptemplates is removed from the dependencies)